tests/lac/vector_type_traits_is_serial_01.cc

```diff
@@ -0,0 +1,75 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 1998 - 2017 by the deal.II authors
```

Comment:
Done. In future, this will be done by a script (I've seen the comment above to late).